### ISSUE TITLE
Sign/text rendering is slow

### ISSUE THREAD

**Issue by SnooComics9452** [jellysquid3/Sodium#350]
I Like Sodium, It Is Very Good.
But A Minecraft Block Makes Still Lagging

That Is Signs. In Some Of Map Metropolis Places Still Lag Because There Have Lot Of Signs
Even Chinese Words Is Most Lag If Type On Signs.
I Suggest To Make Sodium Can No Rendering Signs Behind Blocks But Still Rendering Signs Behind Transparent Blocks Like Glass And Iron Bars

This Feature Can Makes No Lagging In All Maps.
I Suggest This Map For Test And Debug. https://www.planetminecraft.com/project/my-first-minecraft-world-republic-of-flashteens/

Update 20201023 I Also Found Banners Have Same Issue

**Comment by Miros77** [comment]
use v-syns

**Comment by Titaniumtown** [comment]
The capitalization and way this issue is written hurts my head.

**Comment by SnooComics9452** [comment]
> 使用v-syns

This Is Very Dump Way

**Comment by Titaniumtown** [comment]
lol this github issue thread is amazing.

**Comment by 8bit-coder** [comment]
> The capitalization and way this issue is written hurts my head.

I agree, that capitalization is horrible. YOU MIGHT AS WELL START JUST TYPING LIKE THIS.

**Comment by jellysquid3** [comment]
Please find some other issue tracker to mock/insult users who don't have great English skills. This issue is absolutely readable and legitimate.
